registry/api/errcode/errors.go

```diff
@@ -97,25 +101,28 @@ type Error struct {
 
 var _ error = Error{}
 
-// ErrorCode returns the ID/Value of this Error
+// ErrorCode returns the ID/Value of this Error.
 func (e Error) ErrorCode() ErrorCode {
 	return e.Code
 }
 
 // Error returns a human readable representation of the error.
 func (e Error) Error() string {
-	return fmt.Sprintf("%s: %s",
```

Comment:
`ErrorCode` doesn't have an `ID`. They have a string value which represents the serialization of the code. The `ErrorCode` is the ID but we call a Code.